src/data_inclusion/tasks/sources/mes-aides.py

```diff
@@ -0,0 +1,122 @@
+# Imports
+from pathlib import Path
+from datetime import datetime
+from dateutil.parser import parse as dateutil_parse
+import pytz
+import numpy as np
+import pandas as pd
+from pyairtable import Table
+
+GARAGES_SOLIDAIRES_SOURCE_STR = "garages solidaires"
```

Comment:
```suggestion
MES_AIDES_SOURCE_STR = "mes-aides"
```
Pour le moment, le champs `source` identifie le partenaire.

(nb: C'est notamment ce que l'on fait dans pour `itou`. Comme tu peux le voir dans `./dags/main.py` : on moissonne 2 tables différentes (`structures/?type=siae` et `structures/?type=orga`) et on attribut à l'ensemble `source=itou`. Effectivement ça peut être interessant de distinguer les datasets, mais par cohérence je te propose de garder `source=<nom_du_partenaire>` et de changer globalement par la suite)